astroquery/solarsystem/pds/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""
+RingNode
+--------
+
+:author: Ned Molter (emolter@berkeley.edu)
+"""
+
+from astropy import config as _config
+
+
+class Conf(_config.ConfigNamespace):
+    """
+    Configuration parameters for `astroquery.solarsystem.pds`.
+    """
+
+    # server settings
+    pds_server = _config.ConfigItem(
+        "https://pds-rings.seti.org/cgi-bin/tools/viewer3_xxx.pl?", "Ring Node"
+    )
```

Comment:
Does this service have any mirrors? If not then providing a configuration item for the server might not be worth the effort.